concepticondata/conceptlists/Clark-2008-757.tsv

```diff
@@ -0,0 +1,789 @@
+Clark-2008-757-694	694	to kneel	66	KNEEL
+Clark-2008-757-695	695	to crawl	96	CRAWL
+Clark-2008-757-696	696	to enter	749	ENTER
+Clark-2008-757-697	697	to push	1452	PUSH
+Clark-2008-757-698	698	to kick	1135	KICK
+Clark-2008-757-699	699	to throw	1456	THROW
+Clark-2008-757-700	700	to fall	1280	FALL
+Clark-2008-757-701	701	to tie	1917	TIE
+Clark-2008-757-702	702	to wipe	1454	WIPE
+Clark-2008-757-703	703	to hit	1433	HIT
+Clark-2008-757-704	704	to split	1437	SPLIT
+Clark-2008-757-705	705	to cut (hair)	2155	CUT (WITH SCISSORS)
+Clark-2008-757-706	706	to stab	1434	STAB
+Clark-2008-757-707	707	to grind	1033	GRIND
+Clark-2008-757-708	708	to plant	1486	PLANT (SOMETHING)
+Clark-2008-757-709	709	to dig	1418	DIG
+Clark-2008-757-710	710	to bury (a corpse)	1719	BURY
+Clark-2008-757-711	711	to dry (sth.)	1398	DRY
+Clark-2008-757-712	712	to pound (rice)	354	POUND
+Clark-2008-757-713	713	to boil (sth.)	2492	BOIL (SOMETHING)
+Clark-2008-757-714	714	to burn	2102	BURN
+Clark-2008-757-715	715	to work	1086	WORK (ACTIVITY)
+Clark-2008-757-716	716	to play	1413	PLAY
+Clark-2008-757-717	717	to dance	1879	DANCE
+Clark-2008-757-718	718	to shoot	1172	SHOOT
+Clark-2008-757-719	719	to hunt	1435	HUNT
+Clark-2008-757-720	720	to fight	1423	FIGHT
+Clark-2008-757-721	721	to exchange	2509	EXCHANGE
+Clark-2008-757-722	722	to disappear	485	DISAPPEAR
+Clark-2008-757-723	723	to flow	2003	FLOW
+Clark-2008-757-724	724	to float	1574	FLOAT
+Clark-2008-757-725	725	to sink	1088	SINK (DESCEND)
+Clark-2008-757-726	726	to twinkle	2889	TWINKLE
+Clark-2008-757-727	727	to cook (rice)	1100	COOK (SOMETHING)
+Clark-2008-757-728	728	to wash rice	1453	WASH
+Clark-2008-757-729	729	to cut into pieces	2149	CHOP (INTO PIECES)
+Clark-2008-757-730	730	to cut into slices		
+Clark-2008-757-731	731	to chop/mince	2149	CHOP (INTO PIECES)
+Clark-2008-757-732	732	to knead (a dough)	274	KNEAD
+Clark-2008-757-733	733	to pound (garlic)	354	POUND
+Clark-2008-757-734	734	to put in (ingredients)	998	PUT
+Clark-2008-757-735	735	to pour over	1487	POUR
+Clark-2008-757-736	736	to squeeze	1414	SQUEEZE
+Clark-2008-757-737	737	to crack an egg		
+Clark-2008-757-738	738	to boil	792	BOIL (OF LIQUID)
+Clark-2008-757-739	739	to stir fry, to fry	2162	STIR FRY
+Clark-2008-757-740	740	to fry in fat/oil	991	FRY
+Clark-2008-757-741	741	to wash	1453	WASH
+Clark-2008-757-742	742	to write	1672	WRITE
+Clark-2008-757-743	743	to tear	1735	TEAR (SHRED)
+Clark-2008-757-744	744	to talk	2708	TALK
+Clark-2008-757-745	745	to stoke		
+Clark-2008-757-746	746	to soak	2069	SOAK (VERB)
+Clark-2008-757-747	747	to smell	2124	SMELL
+Clark-2008-757-748	748	to slash		
+Clark-2008-757-749	749	to set trap		
+Clark-2008-757-750	750	to scrape	1767	SCRAPE
+Clark-2008-757-751	751	to scold	299	SCOLD
+Clark-2008-757-752	752	to run	1519	RUN
+Clark-2008-757-753	753	to roar	2876	ROAR (OF SEA)
+Clark-2008-757-754	754	to ride horse	367	RIDE
+Clark-2008-757-755	755	to rest	168	REST
+Clark-2008-757-756	756	to remember	707	REMEMBER
+Clark-2008-757-757	757	to pull gr	1455	PULL
```

Comment:
Not sure what the "gr" means but because we have the verb already somewhere else, I'd suggest to delete the mapping.

Reply:
Mei-Shin, can you point to the original source? Is it a PDF, can you check?

Reply:
The PDF can be viewed here : https://inter.payap.ac.th/wp-content/uploads/linguistics_students/Eddie_Clark_Thesis.pdf

Reply:
ok, I will unlink this one

> Not sure what the "gr" means but because we have the verb already somewhere else, I'd suggest to delete the mapping.